spk/rutorrent/src/conf/resource

```diff
@@ -4,5 +4,55 @@
 			"relpath": "etc/php/conf.d/com.synocommunity.rutorrent.ini",
 			"destpath": "/usr/local/etc/php56/conf.d"
 		}]
-    }
+    },
```

Comment:
Curious about these lines... in the developer guide the example just has "php" for the PHP INI as in:

> ### Syntax
> ```
> "php": {
> 	"php-ini": [{
> 		"relpath": "<ini-relpath>",
> 	}, ...],
> 	"fpm-conf": [{
> 		"relpath": "<conf-relpath>",
> 	}, ...]
> }
> ```

I wonder if this is a potential issue since in your `Makefile` you have:

https://github.com/SynoCommunity/spksrc/blob/46d1edbe3380f2f7b84fe4d73d04045ce1a7d898/spk/rutorrent/Makefile#L7